### ISSUE TITLE
ImportError: Error importing plugin "websauna.tests.fixtures": cannot import name '_getpathsec'

### ISSUE THREAD

**Issue by hsk-amboss** [websauna/websauna#131]
with pyramid 1.9.1 running py.test fails

in ../websauna/system/devop/cmdline.py
`from pyramid.paster import bootstrap, setup_logging, _getpathsec
`

_getpathsec seems to not be supported anymore. Installing an older-version of pyramid (< 1.8) fixes the issue

```
Traceback (most recent call last):
  File ".../venv/local/lib/python3.5/site-packages/_pytest/config.py", line 437, in import_plugin
    __import__(importspec)
  File ".../venv/local/lib/python3.5/site-packages/_pytest/assertion/rewrite.py", line 211, in load_module
    py.builtin.exec_(co, mod.__dict__)
  File ".../venv/local/lib/python3.5/site-packages/websauna/tests/fixtures.py", line 23, in <module>
    from websauna.system.devop.cmdline import setup_logging
  File ".../venv/local/lib/python3.5/site-packages/websauna/system/devop/cmdline.py", line 8, in <module>
    from pyramid.paster import bootstrap, setup_logging, _getpathsec
ImportError: cannot import name '_getpathsec'
```

**Comment by Gjallarbru** [comment]
The following code doesn't exist in master for pyramid.paster 1.9, and it does in 1.8 (lines 54 to 61). Any import for _getpathsec will undoubtedly fail:

`def _getpathsec(config_uri, name):
    if '#' in config_uri:
        path, section = config_uri.split('#', 1)
    else:
        path, section = config_uri, 'main'
    if name:
        section = name
return path, section`

**Comment by s4g** [comment]
Just installed ws for the first time. Same error as above when attempting the initial migration from the tutorial:
 ws-alembic -c myapp/conf/development.ini -x packages=all revision --auto -m 'initial migration'

File ".../python3.6/site-packages/websauna/system/devop/cmdline.py", line 8, in <module>
    from pyramid.paster import bootstrap, setup_logging, _getpathsec
ImportError: cannot import name '_getpathsec'

Looks like the whole thing is unusable

**Comment by miohtama** [comment]
We do not currently support Pyramid 1.9, but a patch is coming. Thank you for your patience.

**Comment by ericof** [comment]
Pyramid 1.9 is supported on master.

**Comment by say2neeraj** [comment]
@ericof I am new to pyramid, can you please help - how can i sync this fix in current pyramid 1.9.1.
As currently i was trying to install websauna and postgres and i found this error.
I saw the documentation of pyramid and its updated upto 1.9.1a2.
